rbac/server/api/users.py

```diff
@@ -377,7 +353,7 @@
 
 @USERS_BP.get("api/users/<user_id>/roles/recommended")
 @authorized()
-async def fetch_recommended_roles(request, user_id):
+async def fetch_recommended_roles(request):
```

Comment:
When I run the app and I log in I get this error:
fetch_recommended_roles() got an expected keyword argument 'user_id'. Looks like Phil is passing in the user_id variable into this role. Possibly work with Phil to find out what the user_id is used for and if he could change the front end to not pass in the user_id that should fix this error.

Reply:
Spoke with Phil, he just put out a PR #1333 to remove code calling this unused endpoint

Reply:
It appears PR #1333 have been closed as Phil needs to make more changes. I'll check this PR again once he has merged his changes on the UI side.